core/sodasql/common/yaml_helper.py

```diff
@@ -11,3 +11,23 @@ def parse_yaml(yaml_str: str, description: str = None):
             return yaml.load(yaml_str, Loader=yaml.SafeLoader)
         except Exception as e:
             logging.error(f'Parsing YAML failed: {str(e)}: ({description if description else yaml_str})xWW4')
+
+    @staticmethod
+    def validate_numeric_value(column_name, key, value):
+        if value is None:
+            logging.info(f'There is no {key} key of this column name {column_name} to be validated')
+        elif value is isinstance(value, int):
+            logging.error(f'{column_name} could not be parsed: {key}-{value} is not of a numeric type.')
+            raise Exception(f'{column_name} could not be parsed: {key}-{value} is not of a numeric type.')
+        else:
+            return value
+
+    @staticmethod
+    def validate_array_value(column_name, key, value):
```

Comment:
maybe name this `validate_list_value` ? Since python nomenclature prefers `list` than `array`?

Reply:
you're right, sure!